### PR TITLE
files

### PR DESCRIPTION
**Pull request by melvicaballero**
begin

### CODE REVIEW

File: DevOps/Vagrantfile [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    os.password           = ENV['OS_PASSWORD']
+    os.tenant_name        = ENV['OS_TENANT_NAME']
+    os.project_name       = ENV['OS_PROJECT_NAME']
+    os.keypair_name       = ENV['OS_KEY_PAIR_NAME']
+    os.region             = ENV['OS_REGION_NAME']
+    os.image              = ENV['OS_IMAGE']
+    override.vm.synced_folder '.', '/vagrant', disabled: true
+  end
+  config.vm.define 'server-1' do |s|
+    #s.vm.provision "docker"
```
Comment (fercho-ayala) [inline DevOps/Vagrantfile]: Remove unused lines.

File: DevOps/Vagrantfile [hunk 2]
```diff
@@ -0,0 +1,34 @@
+require 'vagrant-openstack-provider'
+Vagrant.configure('2') do |config|
+  config.ssh.username = 'ubuntu'
+  config.ssh.private_key_path = "C:\\Users\\Administrator\\Desktop\\Pardeclaves\\wincmmv.pem"
```
Comment (fercho-ayala) [inline DevOps/Vagrantfile]: You need to use an environment variable for this path.

**Review by fercho-ayala — CHANGES_REQUESTED** [review]
No more comments.

File: DevOps/Vagrantfile (hunk 1 above)
Comment (melvicaballero) [reply DevOps/Vagrantfile]: fixed.

File: DevOps/Vagrantfile (hunk 2 above)
Comment (melvicaballero) [reply DevOps/Vagrantfile]: Variable OS_PRIVATE_KEY_PATH is created on environment file.

**Review by fercho-ayala — APPROVED** [review]
Looks good!

**Comment by fercho-ayala** [comment]
Please merge you PR.

**Pull request merged**